include/mbedtls/ssl.h

```diff
@@ -338,11 +338,15 @@
 #define MBEDTLS_SSL_SRV_CIPHERSUITE_ORDER_SERVER  0
 
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
-#if defined(MBEDTLS_SHA384_C)
+#if ( !defined(MBEDTLS_USE_PSA_CRYPTO) && \
```

Comment:
Sorry for missing this earlier: this is specific to TLS 1.3 (not shared with TLS 1.2) so it does not obey `MBEDTLS_USE_PSA_CRYPTO` and the correct test macro is `PSA_WANT_ALG_SHA_xxx`.